src/app/lib/utilities/setCookie/index.jsx

```diff
@@ -13,10 +14,29 @@ export const getCookieDomain = domain => {
   return domain;
 };
 
-const setCookie = (name, value, expires = COOKIE_EXPIRY) =>
-  Cookie.set(name, value, {
+const setCookie = ({
+  name,
+  value,
+  expires = COOKIE_EXPIRY,
+  sameSite = SAMESITE_DEFAULT,
+}) => {
+  const isHttps = window.location.protocol === 'https:';
+
+  // Modern browsers default sameSite value to Lax
+  // Setting sameSite='None' allows cookies will be sent in all contexts, i.e sending cross-origin is allowed.
+  // Setting sameSite='Strict' allows cookies to only set in first-party context
+  const sameSiteValues = {
+    Lax: 'Lax',
+    Strict: 'Strict',
+    None: isHttps ? 'None' : undefined, // SameSite=None can only be added to cookie when the page is https
```

Comment:
Aren't all pages `https`?

Reply:
I imagine the idea here is that `setCookie()` _can_ be used in all instances. As you've suggested, it will probably only be used to set secure cookies, but this logic should probably exist to prevent setting of invalid cookies.